ephios/core/signals.py

```diff
@@ -49,6 +48,16 @@
 If all forms are valid, `save` will be called on your form.
 """
 
+register_notification_types = Signal()
```

Comment:
@felixrindt can you help me here? I tried this with PluginSignal(), but as I'm accessing the signal already while migrating, the global preferences have not been set up at that point and everything breaks (have a look at https://github.com/ephios-dev/ephios/runs/2107604091). It works this way, do you think that's okay?

Reply:
Where is this used in migrations? Can you avoid that?